packages/views/signoff.py

```diff
@@ -53,6 +55,36 @@ def signoff_package(request, name, repo, arch, revoke=False):
             return render(request, '403.html', status=403)
         signoff, created = Signoff.objects.get_or_create_from_package(
                 package, request.user)
+        signoffs = Signoff.objects.for_package(package).filter(
+                revoked__isnull=True)
+
+        signed_off_by = []
+        for s in signoffs:
+            signed_off_by.append(User.objects.get(id=s.user_id))
+
+        if  signoffs.count() >= spec.required:
+            packager = package.packager
+            if not packager:
+                toemail = [package.packager_str.split('<')[1].replace('>','')]
+            else:
+                toemail = [packager.email]
+            subject = '%s package [%s %s] approved' % \
+                    (package.repo.name, package.pkgname,
+                     package.full_version)
```

Comment:
we should support f-strings now

Reply:
Thanks for the heads up! I was unaware of the f-strings functionality, I'm glad you told me about them.